lib/utils/registry/registry_windows.go

```diff
@@ -0,0 +1,85 @@
+//go:build windows
```

Comment:
This is not necessary since the filename ends with `_windows.go`

https://pkg.go.dev/go/build#hdr-Build_Constraints